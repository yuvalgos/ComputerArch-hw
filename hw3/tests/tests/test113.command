./dflow_calc tests/test113.dat tests/test113.in p179 d153 p88 d29 d1 d76 p22 d176 p122 p162 p99 p110 d47 d190 p131 d160 p199 p30 p44 p30 p112 p114 p107 d61 p8 d66 d70 d56 d65 d20 d87 p35 p155 d57 d176 p150 p147 d180 d170 d98 p127 d130 p128 d80 p11 p14 d84 d138 d157 p91 d37 d186 d169 d84 d109 d95 p95 d187 d58 p166 d6 p157 p106 p119 d42 d34 p35 d78 d76 p30 d21 d100 p74 d192 p0 p150 d12