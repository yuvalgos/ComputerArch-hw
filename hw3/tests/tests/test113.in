4 28 21 26
1 29 11 8
3 24 28 1
6 13 8 13
2 11 25 29
0 5 29 2
7 2 16 31
2 0 10 20
6 6 11 10
8 4 30 16
6 22 15 22
8 20 7 20
5 30 11 22
1 9 1 26
4 18 10 30
8 22 12 11
3 17 13 23
4 31 19 1
5 7 29 1
0 1 25 24
8 21 23 4
5 16 13 21
5 30 17 10
4 7 20 14
5 0 17 12
4 1 20 6
7 24 4 4
8 31 27 14
7 21 27 20
7 25 24 30
6 28 24 4
8 19 8 4
3 20 15 22
4 1 13 7
1 16 8 13
7 4 22 26
6 10 22 0
7 30 8 30
0 28 29 5
2 6 18 4
1 17 29 13
2 9 24 10
8 23 26 29
0 13 11 24
7 4 18 15
1 20 12 22
7 8 12 26
0 23 23 27
0 27 22 21
7 30 24 1
4 14 19 5
2 15 0 9
7 17 23 26
4 15 1 17
5 5 15 2
1 3 22 25
8 12 21 13
3 31 16 25
3 26 2 25
6 30 12 22
7 24 20 14
3 31 25 22
1 23 12 12
0 21 5 13
1 31 12 3
8 24 8 5
1 19 22 3
5 22 28 31
7 27 22 5
1 22 17 21
4 21 25 25
1 22 2 10
7 18 18 0
3 14 21 14
5 29 10 14
3 6 23 27
1 10 22 10
6 26 17 30
5 6 19 22
6 5 14 2
1 9 12 29
3 27 10 12
4 8 26 2
1 1 17 20
7 21 12 11
7 26 11 19
3 23 7 13
8 30 22 22
5 28 30 2
6 10 28 20
3 5 25 5
8 2 29 3
4 17 19 24
5 15 15 16
4 14 11 14
4 3 20 22
6 24 4 16
4 31 4 23
3 28 27 18
4 6 23 24
8 22 19 23
6 8 18 24
8 0 18 2
8 28 30 29
7 20 28 16
1 14 17 27
4 27 29 31
6 2 14 31
1 18 15 5
8 4 25 31
8 3 18 21
7 1 11 2
0 9 20 6
3 21 31 20
0 10 12 14
0 6 11 18
0 10 24 14
8 10 5 26
7 14 1 19
2 13 19 26
2 16 13 0
5 20 7 23
6 16 8 18
6 23 8 22
3 16 17 25
5 7 24 13
7 31 13 4
8 23 7 12
2 9 18 28
0 31 2 12
8 8 16 17
6 28 12 19
8 12 31 1
3 0 25 18
3 15 20 6
8 10 25 19
5 7 9 0
6 0 25 9
3 22 4 3
5 3 19 30
7 29 25 7
1 8 15 13
6 7 3 11
8 1 21 4
8 4 23 1
7 7 19 2
8 28 25 3
8 9 9 19
6 9 24 0
1 12 4 0
7 28 11 27
3 9 24 13
5 23 31 16
0 28 21 18
2 17 7 25
8 30 24 30
3 30 30 20
2 17 12 18
1 1 30 4
2 22 23 28
3 5 18 19
5 13 3 23
3 21 27 1
3 26 18 29
0 12 17 31
6 19 19 16
8 19 21 2
4 22 17 2
5 20 0 21
1 10 19 6
6 28 17 21
6 18 15 25
1 1 29 24
7 10 1 28
5 15 20 15
1 11 16 2
8 8 27 29
2 20 18 13
2 4 2 7
4 25 0 8
1 23 1 22
5 25 0 26
0 29 23 23
3 29 4 23
2 19 3 19
3 8 22 8
7 7 30 11
8 1 27 6
7 18 11 29
8 26 21 0
2 30 29 6
7 30 3 11
7 25 8 28
7 26 11 3
8 15 11 24
5 22 24 1
1 1 21 8
0 19 18 20
2 2 9 26
1 7 3 0
