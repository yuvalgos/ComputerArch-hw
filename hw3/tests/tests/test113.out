Reading the opcodes latency info from tests/test113.dat ... Got latency for 9 opcodes
Reading the program file tests/test113.in ... Found 200 instructions
getProgDepth()==237
getDepDepth(179)==175
getInstDeps(153)=={113,108}
getDepDepth(88)==80
getInstDeps(29)=={26,22}
getInstDeps(1)=={-1,-1}
getInstDeps(76)=={71,36}
getDepDepth(22)==24
getInstDeps(176)=={106,140}
getDepDepth(122)==106
getDepDepth(162)==190
getDepDepth(99)==77
getDepDepth(110)==106
getInstDeps(47)=={42,-1}
getInstDeps(190)=={183,115}
getDepDepth(131)==67
getInstDeps(160)=={108,66}
getDepDepth(199)==137
getDepDepth(30)==42
getDepDepth(44)==41
getDepDepth(30)==42
getDepDepth(112)==134
getDepDepth(114)==108
getDepDepth(107)==108
getInstDeps(61)=={29,15}
getDepDepth(8)==8
getInstDeps(66)=={15,55}
getInstDeps(70)=={29,29}
getInstDeps(56)=={28,43}
getInstDeps(65)=={46,54}
getInstDeps(20)=={-1,9}
getInstDeps(87)=={71,71}
getDepDepth(35)==31
getDepDepth(155)==80
getInstDeps(57)=={34,29}
getInstDeps(176)=={106,140}
getDepDepth(150)==103
getDepDepth(147)==120
getInstDeps(180)=={172,167}
getInstDeps(170)=={157,162}
getInstDeps(98)=={81,72}
getDepDepth(127)==104
getInstDeps(130)=={124,92}
getDepDepth(128)==115
getInstDeps(80)=={56,74}
getDepDepth(11)==0
getDepDepth(14)==30
getInstDeps(84)=={56,4}
getInstDeps(138)=={109,110}
getInstDeps(157)=={149,108}
getDepDepth(91)==92
getInstDeps(37)=={-1,22}
getInstDeps(186)=={156,175}
getInstDeps(169)=={166,115}
getInstDeps(84)=={56,4}
getInstDeps(109)=={29,97}
getInstDeps(95)=={45,71}
getDepDepth(95)==57
getInstDeps(187)=={106,115}
getInstDeps(58)=={6,29}
getDepDepth(166)==197
getInstDeps(6)=={-1,-1}
getDepDepth(157)==190
getDepDepth(106)==92
getDepDepth(119)==86
getInstDeps(42)=={-1,1}
getInstDeps(34)=={-1,3}
getDepDepth(35)==31
getInstDeps(78)=={66,71}
getInstDeps(76)=={71,36}
getDepDepth(30)==42
getInstDeps(21)=={3,20}
getInstDeps(100)=={66,86}
getDepDepth(74)==79
getInstDeps(192)=={185,170}
getDepDepth(0)==0
getDepDepth(150)==103
getInstDeps(12)=={4,10}
